### ISSUE TITLE
Missing support for loading additional support assemblies during avro create

### ISSUE THREAD

**Issue by digitalpacman** [ch-robinson/dotnet-avro#95]
If you are using extra libraries around the models that use attributes, dotnet avro create -a ASSEMBLY -t TYPE throws assembly loading exceptions.  For example if you are also using these models with the System.Text.Json serializer and have properties marked with [JsonPropertyName], then when running the avro create command will throw could not load file or assembly error.  What I think is necessary is to make the -a flag a list value, allowing the assemblies to be loaded.  That, or add support for loading a csproj file instead which would include all the nuget includes, etc.  Or allow for setting the current working directory.  I'm guessing that assemblies are being loaded from the global install path of dotnet-avro.

In the steps to reproduce the app wants to load System.Text.Json version 4.0.1.2. This fails.
On my machine not using the nuget install sets the assembly to require System.Text.Json version 4.0.1.0. This Succeeds.
I don't know why but project references load fine.

Steps to reproduce:

1. Create new dotnet core console application
2. Use nuget to add System.Text.Json version 4.7.2
3. Create a dummy class to be the avro model
4. Add [JsonPropertyName()] attribute to a property of the model
5. Build
5. Run the command to build a schema from that assembly and type: dotnet-avro -a assembly -t type 
6. The error will reproduce

**Comment by dstelljes** [comment]
Definitely on board with making `-a` a list value. The [library we use for parsing args](https://github.com/commandlineparser/commandline) ships with support for that, so it should be an easy change.

It would also be neat if we could specify a csproj (or, if the command is run in a project directory, use that one?). I remember looking into that when we initially implemented the CLI and the .NET Core tooling wasn't quite there yet; would be good to revisit that.

**Comment by digitalpacman** [comment]
👍  What I'm doing to move forward is using interfaces to ensure type similarity and building the schemas off raw POCOS with no dependencies at all

**Comment by dstelljes** [comment]
Support for multiple `-a` values will ship in a patch release following #96. Once that lands, this syntax should work:

```bash
$ dotnet avro create -a System.Text.Json path/to/Example.dll -t Example.Thing
```

**Comment by dstelljes** [comment]
Released (finally) with 7.0.1.
